board.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+
+class Board:
+    _characters = {
+        ' ': 0,  # free space
+        '@': 1,  # player
+        '#': 2,  # wall
+        '$': 3,  # box
+        '.': 4,  # goal
+    }
+
+    def __init__(self, filename):
+
+        with open(filename, 'r') as f:
+
+            read_data = f.read()
+            self._lines = read_data.split("\n")
+
+            self.width = max(list(map(lambda x: len(self._lines[x]), range(len(self._lines)))))
+            self.height = len(self._lines)
+
+            self.board = np.zeros((self.width, self.height), dtype=int)
```

Comment:
@jcness216 I changed this to np.zeros because np.empty gave really weird big numbers.  or was there a reason you had it as np.empty?